File: 8-kyu/crash-override/python/solution.py
def alias_gen(f_name, l_name):
    return f'{FIRST_NAME[f_name[0].upper()]} {SURNAME[l_name[0].upper()]}' if f_name[0].isalpha() and l_name[0].isalpha() else 'Your name must start with a letter from A - Z.'

File: 8-kyu/crash-override/javascript/solution.js
const aliasGen = (first, last) => {
  const firstLetterAlpha = /^[a-z]/i;
  if (firstLetterAlpha.test(first) && firstLetterAlpha.test(last)) {
    return `${firstName[first[0].toUpperCase()]} ${surname[last[0].toUpperCase()]}`;
  }
  return "Your name must start with a letter from A - Z.";
};
